src/Middleware/HttpLogging/src/HttpLoggingOptions.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Net.Http.Headers;
+
+namespace Microsoft.AspNetCore.HttpLogging
+{
+    /// <summary>
+    /// Options for the <see cref="HttpLoggingMiddleware"/>
+    /// </summary>
+    public sealed class HttpLoggingOptions
+    {
+        /// <summary>
+        /// Fields to log for the Request and Response. Defaults to logging request and response properties and headers.
+        /// </summary>
+        public HttpLoggingFields LoggingFields { get; set; } = HttpLoggingFields.RequestPropertiesAndHeaders | HttpLoggingFields.ResponsePropertiesAndHeaders;
+
+        /// <summary>
+        /// Request header values that are allowed to be logged.
+        /// </summary>
+        /// <remarks>
+        /// If a request header is not present in the <see cref="RequestHeaders"/>,
+        /// the header name will be logged with a redacted value.
+        /// </remarks>
```

Comment:
```suggestion
        /// <p>
        /// If a request header is not present in the <see cref="RequestHeaders"/>,
        /// the header name will be logged with a redacted value.
        /// </p>
        /// </summary>
```

No remarks. This is part of the summary

Reply:
Do we have any documentation on how to write great API/doc comments? I've seen remarks used elsewhere/I personally have used them elsewhere.